app/graph/mutations/search_upload_mutations.rb

```diff
@@ -1,14 +1,12 @@
 module SearchUploadMutations
-  SearchUpload = GraphQL::Relay::Mutation.define do
-    name 'SearchUpload'
+  class SearchUpload < Mutations::BaseMutation
```

Comment:
The graphql_name by default is SearchUpload because it pulls it from the class (docs on that [here](https://graphql-ruby.org/type_definitions/directives.html#custom-name)), as such we only need to specify the graphql_name if the name we need can't be automatically generated from the class name.

I thought about changing all of our class names to match the expected graphql_name, but it didn't really match our module structure so left it as-is except for the few cases where we already have a class name that matches the graphql_name